### PR TITLE
Leaderboard hit api

### PR DESCRIPTION
**Pull request by Chimoneg27**
### For this feature I did the following

- Made sure to remove linter errors
- Used the post method with async await to send user inputs to the leaderboard API
- Used the get method with async await and fetch to get user inputs from the leaderboard API
- Made use of arrow functions
- Organized my code into modules
- Removed hardcoded HTML

**Notice: I would really need some assistance on this error**

https://github.com/Chimoneg27/Leaderboard-With-API/assets/106396944/1338a0fa-4a34-40a4-9246-f2489cf52e80

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,2 +1,33 @@
...
+
+  const userInput = document.getElementById('playerName');
+  const scoreInput = document.getElementById('points');
+  const user = userInput.value;
+  const score = scoreInput.value;
+  const newData = new Data(user, score);
+
+  await submitScore(newData);
+  document.getElementById('playerName').value = '';
+  document.getElementById('points').value = '';
```
Comment (Tufahel) [inline src/index.js]: - [ ] Kindly check the `index.html` and add `Id` carefully. the `ID` name inside `getElementById()` should be `scores` not `points`. Fix it kindly using this `  document.getElementById('scores').value = '';` I am unable to submit the `Name` and `Score` for this error. Kindly fix this.

File: src/index.js [hunk 2]
```diff
@@ -1,2 +1,33 @@
...
+
+const formData = document.getElementById('form-box');
+const refreshBTN = document.getElementById('refresh-btn');
+const scoreList = document.getElementById('table');
+
+formData.addEventListener('submit', async (e) => {
+  e.preventDefault();
+
+  const userInput = document.getElementById('playerName');
+  const scoreInput = document.getElementById('points');
```
Comment (Tufahel) [inline src/index.js]: - [ ] Kindly check the `index.html` and add `Id` carefully. the `ID` name inside `getElementById()` should be `scores` not `points`. Fix it kindly using this `const scoreInput = document.getElementById('scores');` I am unable to submit the `Name` and `Score` for this error. Kindly fix this.

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @Chimoneg27 :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ The app is working fine.
✔️ Used async, await, and arrow functions.

## Required Changes ♻️

- [ ]  Kindly use the correct information in a readme file. Check the title and details of your Readme file. it is not correct. Your project is `Leaderboard` but you have written `Awesome-Books`.
- Screenshot from 2023-09-01 03-41-53

- [ ] Same here the wrong information in readme. Your project name is `Leaderboard-With-API`. So it should be `cd Leaderboard-With-API`. Kindly fix this.
- image

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

✔️ The app is working fine.
✔️ Used async, await, and arrow functions.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**